src/main/java/gift/auth/GetLoginInfo.java

```diff
@@ -0,0 +1,15 @@
+package gift.auth;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * 로그인 정보를 가져오기 위한 어노테이션
+ */
+@Target(ElementType.PARAMETER)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface GetLoginInfo {
```

Comment:
Get과 같은 동사형은 메서드의 네이밍에 사용되므로 LoginInfo로 줄이셔도 될 것 같아요ㅎㅎ
인증된 사용자라는 의미에서 Authenticate라고 해도 괜찮을 것 같네요!
```suggestion
public @interface Authenticate {
```

Reply:
참고로 이러한 클래스, 메서드, 변수 등의 네이밍 변경은 Shift + F6을 누르면 연관된 모든 곳에 일괄변경됩니다!
혹시 모르셨다면 참고 부탁드려요ㅎㅎ

Reply:
감사합니다! 네이밍 변경 하겠습니다!